app/models/site_config.rb

```diff
@@ -103,10 +103,6 @@ class SiteConfig < RailsSettings::Base
         type: :string,
         default: proc { URL.local_image("mascot.png") },
         validates: { url: true }
-  field :mascot_image_description, type: :string, default: "The community mascot"
-  field :mascot_footer_image_url, type: :string, validates: { url: true }
-  field :mascot_footer_image_width, type: :integer, default: 52
-  field :mascot_footer_image_height, type: :integer, default: 120
```

Comment:
should we delete these explicitly with the data update script?

Reply:
I'll do all of that in the follow-up PR where I also remove the `settings_mascot` table.